NDTensors/src/linearalgebra/linearalgebra.jl

```diff
@@ -400,6 +400,7 @@ function qx(qx::Function, T::DenseTensor{<:Any,2}; kwargs...)
   Qinds = IndsT((ind(T, 1), q))
   Xinds = IndsT((q, ind(T, 2)))
   QM = convert(typeof(XM), QM)
+  QM = convert(typeof(XM), QM)
```

Comment:
Seems to just be a repeat of the line above.

Reply:
There seems to be a conversion issue related to `UnifiedMemory` buffer in CUDA. The first `convert` takes the `packedQR` type which is a CPU wrapper to `CuArray{ElT, N, DeviceBuffer}` and the second call converts it to `UnifiedBuffer`. I have tried using `adapt` and calling the `CuArray` constructor but neither seem to create the vector with the correct memory without the second call.

Reply:
Reminder to leave a comment about why this is here and that it should be removed once the issue is fixed in CUDA.jl.

Reply:
I added a comment here, thanks!